server/pulp/server/async/app.py

```diff
@@ -179,6 +180,11 @@ def initialize_worker(sender, instance, **kwargs):
         get_resource_manager_lock(sender)
 
 
+@worker_process_init.connect
+def reconnect_services(sender, **kwargs):
```

Comment:
Add a simple docstring here explaining the purpose of this, perhaps linking to the issue.  See ```initialize_workers()``` for reference but no need to be so comprehensive.